sdks/python/apache_beam/io/external/jdbc.py

```diff
@@ -0,0 +1,254 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
```

Comment:
Please move this to io module directly. "apache_beam/io/jdbc.py"

Reply:
Ok, done